.circleci/config.yml

```diff
@@ -191,6 +191,7 @@ jobs:
         type: string
     docker:
       - image: cimg/base:stable-18.04
+    resource_class: medium+
```

Comment:
Please add a comment explaining this (or with a link to an issue explaining it).

Reply:
will do :)